src/devices/lumi.ts

```diff
@@ -4579,6 +4579,17 @@ export const definitions: DefinitionWithExtend[] = [
         model: "TH-S04D",
         vendor: "Aqara",
         description: "Climate Sensor W100",
+        fromZigbee: [lumi.fromZigbee.w100_0844_req, lumi.fromZigbee.pmtsd_from_w100],
+        toZigbee: [lumi.toZigbee.pmtsd_to_w100, lumi.toZigbee.thermostat_mode],
+        exposes: [
+            e.action(["W100_PMTSD_request"]).withDescription("W100 Requesting PMTSD Data via 08000844 Request"),
+            e.text("PMTSD_from_W100_Data", ea.STATE).withDescription("Timestamp+Most Recent PMTSD Values Sent by W100"),
```

Comment:
This and the action above is very unclear to me, can you extend the description on what it does?